tipboard/tiles/line_chart.js

```diff
@@ -1,6 +1,67 @@
 /*jslint browser: true, devel: true*/
 /*global WebSocket: false, Tipboard: false*/
 
+function simplifyLineData(series, tolerancy, data_points_limit, max_simplifying_steps, simplify_step_multiplicator) {
+    data_points_limit = data_points_limit || 50; // we will TRY to achieve lower number of data points than this
+    max_simplifying_steps = max_simplifying_steps || 5;
+    simplify_step_multiplicator = simplify_step_multiplicator || 1.5;
+    
+    var simplify_data = new Array();
+    var return_data = new Array();
+
+    for(var serie = 0; serie < series.length; serie++) {
```

Comment:
unfortunately, there is no such word in english as serie (series is both singular and plural).